packages/@headlessui-react/src/components/menu/menu.tsx

```diff
@@ -572,8 +573,10 @@ type MenuItemPropsWeControl =
   | 'role'
   | 'tabIndex'
   | 'aria-disabled'
+  | 'onPointerEnter'
   | 'onPointerLeave'
   | 'onPointerMove'
+  | 'onMouseEnter'
```

Comment:
We probably have to update these types anyway, but the idea behind these was to not provide autocompletion for those since we are overriding them regardless of what the user provided.

But, now that we merge function props (onXXX), we can probably remove all of those `onXXX` ones. 

Just an fyi

Reply:
Oh sweet